front/src/pods/trainer/trainer.container.tsx

```diff
@@ -54,9 +54,11 @@ export const TrainerContainer = () => {
   }, []);
 
   const handleAppendTrainerText = (trainerText: string): void => {
+    const separation = '\n\n*********************************\n';
```

Comment:
It would be a good idea to extract this to a const, maybe located at core/consts or something like that

On the other hand for the sake of code readiblity we could go for something like:

```typescript
const AppendLineSeparator = (text : string)  : string => 
   `${lineseparator}${text}`;

const sendTrainerTextToServer(text : string) : void => 
    socket.emit(SocketOuputMessageLiteral.MESSAGE, {
      type: SocketEmitMessageTypes.TRAINER_APPEND_TEXT,
      payload: text,
    });

const handleAppendTrainerText = (trainerText: string) => {
  const finalText = addLineSeparator(trainerText);
  sendTrainerTextToServer(finalText);
} 
```